### PR TITLE
Add the header file

### PR DESCRIPTION
**Pull request by stefanocampanella**

### CODE REVIEW

File: include/ljmd.h [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+#define __LJMD__
+
+
+/* 
+ * simple lennard-jones potential MD code with velocity verlet.
+ * units: Length=Angstrom, Mass=amu; Energy=kcal
+ *
+ * baseline c version.
+ *
+ * latest revision: Stefano Campanella, 02/12/2020
```
Comment (lleon95) [inline include/ljmd.h]: By experience, adding the latest version is kind of useless. You can forget updating it.

File: include/ljmd.h [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+/**
+ * @brief conversion constant: m*v^2 in kcal/mol
+ */
+const double mvsq2e = 2390.05736153349;
+
+/**
+ * @brief helper function: read a line and then return
+ *  the first string with whitespace stripped off
+ */
+static int get_a_line(FILE *fp, char *buf);
```
Comment (lleon95) [inline include/ljmd.h]: Remove static keyword

File: include/ljmd.h [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+
+/**
+ * @brief helper function: zero out an array
+ */
+static void azzero(double *d, const int n);
```
Comment (lleon95) [inline include/ljmd.h]: Remove static keyword

File: include/ljmd.h [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+
+/**
+ * @brief helper function: apply minimum image convention
+ */
+static double pbc(double x, const double boxby2);
```
Comment (lleon95) [inline include/ljmd.h]: Remove static keyword

File: include/ljmd.h [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+
+/**
+ * @brief compute kinetic energy
+ */
+static void ekin(mdsys_t *sys);
```
Comment (lleon95) [inline include/ljmd.h]: Remove static keyword

File: include/ljmd.h [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+
+/**
+ * @brief compute forces
+ */
+static void force(mdsys_t *sys);
```
Comment (lleon95) [inline include/ljmd.h]: Remove static keyword

File: include/ljmd.h [hunk 7]
```diff
@@ -0,0 +1,90 @@
...
+
+/**
+ * @brief velocity verlet
+ */
+static void velverlet(mdsys_t *sys);
```
Comment (lleon95) [inline include/ljmd.h]: Remove static keyword

File: include/ljmd.h [hunk 8]
```diff
@@ -0,0 +1,90 @@
...
+
+/**
+ * @brief propagate velocities
+ */
+void propagate_velocity(mdsys_t *sys, int i);
+
+/**
+ * @brief append data to output.
+ */
+static void output(mdsys_t *sys, FILE *erg, FILE *traj);
```
Comment (lleon95) [inline include/ljmd.h]: Remove static keyword

File: include/ljmd.h [hunk 9]
```diff
@@ -0,0 +1,90 @@
...
+/**
+ * @brief conversion constant: m*v^2 in kcal/mol
+ */
+const double mvsq2e = 2390.05736153349;
+
+/**
+ * @brief helper function: read a line and then return
+ *  the first string with whitespace stripped off
+ */
+int get_a_line(FILE *fp, char *buf);
```
Comment (lleon95) [inline include/ljmd.h:48]: Add <stdio.h>

**Review by lleon95 — CHANGES_REQUESTED** [review]

File: include/ljmd.h [hunk 10]
```diff
@@ -0,0 +1,91 @@
+#include <stdio.h>
```
Comment (lleon95) [inline include/ljmd.h]: It is outside of the header guard

**Review by lleon95 — CHANGES_REQUESTED** [review]

**Review by lleon95 — APPROVED** [review]

File: include/ljmd.h [hunk 11]
```diff
@@ -0,0 +1,90 @@
...
+  double *rx, *ry, *rz;
+  double *vx, *vy, *vz;
+  double *fx, *fy, *fz;
+};
+typedef struct _mdsys mdsys_t;
+
+/**
+ * @brief boltzman constant in kcal/mol/K
+ */
+const double kboltz = 0.0019872067;
```
Comment (lleon95) [inline include/ljmd.h]: This constant will lead to conflicts when linking. Please, do:

1. Use: 

```c
extern const double kboltz;
extern const double mvsq2e;
```

2. Create a file named: "constants.c" with the following contents:

```c
#include "../include/ljmd.h"

const double kboltz = 0.0019872067;
const double mvsq2e = 2390.05736153349;
```

**Review by lleon95 — CHANGES_REQUESTED** [review]

**Review by lleon95 — APPROVED** [review]

**Review by mattiacmhpc — APPROVED** [review]

**Pull request merged**